src/types.rs

```diff
@@ -154,135 +137,3 @@ impl CacheOptions {
         self.cache_immutable_data
     }
 }
-
-// #[cfg(test)]
-// #[allow(deprecated)]
-// mod test {
-//     extern crate cbor;
-//     use super::*;
-//     use sodiumoxide::crypto;
-//     use std::cmp;
-//     use rustc_serialize::{Decodable, Encodable};
-//     use test_utils::Random;
-//     use public_id::PublicId;
-//     use authority::Authority;
-//     use NameType;
-//     use name_type::closer_to_target;
-//     use sodiumoxide::crypto::sign;
-//     use utils;
-//
-//     fn test_object<T>(obj_before : T) where T: for<'a> Encodable + Decodable + Eq {
-//       let mut e = cbor::Encoder::from_memory();
-//       e.encode(&[&obj_before]).unwrap();
-//       let mut d = cbor::Decoder::from_bytes(e.as_bytes());
-//       let obj_after: T = d.decode().next().unwrap().unwrap();
-//       assert_eq!(obj_after == obj_before, true)
-//     }
-//
-//     #[test]
-//     fn test_authority() {
-//       test_object(Authority::ClientManager(Random::generate_random()));
-//       test_object(Authority::NaeManager(Random::generate_random()));
-//       test_object(Authority::NodeManager(Random::generate_random()));
-//       test_object(Authority::ManagedNode);
-//       test_object(Authority::Client(sign::gen_keypair().0));
-//       test_object(Authority::Unknown);
-//     }
-//
-//     #[test]
-//     fn test_destination_address() {
-//       test_object(DestinationAddress::Direct(Random::generate_random()));
-//     }
-//
-//     #[test]
-//     fn test_source_address() {
-//         test_object(SourceAddress::Direct(Random::generate_random()));
-//     }
-//
-//     #[test]
-//     fn serialisation_public_id() {
-//         let obj_before = PublicId::generate_random();
-//
-//         let mut e = cbor::Encoder::from_memory();
-//         e.encode(&[&obj_before]).unwrap();
-//
-//         let mut d = cbor::Decoder::from_bytes(e.as_bytes());
-//         let obj_after: PublicId = d.decode().next().unwrap().unwrap();
-//         assert_eq!(obj_before, obj_after);
-//     }
-//
-//     #[test]
-//     fn test_calculate_relocated_name() {
```

Comment:
this test should move to id.rs if it is not already there, or utils if I remember correct where this function is currently living